SwiftVoiceCallKitQuickstart/ViewController.swift

```diff
@@ -228,7 +228,7 @@ class ViewController: UIViewController, PKPushRegistryDelegate, TVONotificationD
             NSLog("Call disconnected")
         }
         
-        performEndCallAction(uuid: call.uuid)
+        performEndCallAction(uuid: call.uuid!)
```

Comment:
Force unwrapping `call.uuid` is dangerous given our nullable `TVOCall.uuid` property. However, you could preface the `performEndCallAction` lines with a comment stating that:

1. performVoiceCall mutates `call.uuid` by setting it to a non-null value.
2. performOutgoingVoiceCall produces a call with a valid UUID.

This is something we will improve in a future beta.